phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -2040,6 +2046,119 @@ public function format_itemizedlist($open, $name, $attrs, $props) {
         return '</ul>';
     }
 
+    public function format_simplelist($open, $name, $attrs, $props) {
+        if ($open) {
+            $this->cchunk["simplelist"]["type"] = $attrs[Reader::XMLNS_DOCBOOK]["type"] ?? "";
+            $this->cchunk["simplelist"]["columns"] = $attrs[Reader::XMLNS_DOCBOOK]["columns"] ?? 1;
+
+            if ($this->cchunk["simplelist"]["columns"] < 1) {
+                $this->cchunk["simplelist"]["columns"] = 1;
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "inline") {
+                return '<span class="' . $name . '">';
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "vert"
+                || $this->cchunk["simplelist"]["type"] === "horiz") {
+                return '<table class="' . $name . '">' . "\n" . str_repeat(" ", $props["depth"] + 1) . "<tbody>\n";
+            }
+
+            return '<ul class="' . $name . '">';
+        }
+
+        $list = match ($this->cchunk["simplelist"]["type"]) {
+            "inline" => $this->format_inline_simplelist(),
+            "horiz"  => $this->format_horizontal_simplelist($props["depth"]),
+            "vert"   => $this->format_vertical_simplelist($props["depth"]),
+            default  => "</ul>",
+        };
+
+        $this->cchunk["simplelist"] = $this->dchunk["simplelist"];
+
+        return $list;
+    }
+
+    private function format_inline_simplelist() {
+        return implode(", ", $this->cchunk["simplelist"]["members"]) . '</span>';
+    }
+
+    private function format_horizontal_simplelist($depth) {
+        return $this->chunkReduceTable(
+            $this->processTabular(
+                $this->cchunk["simplelist"]["members"],
+                $this->cchunk["simplelist"]["columns"],
+                $depth),
+            $this->cchunk["simplelist"]["columns"],
+            $depth
+        );
+    }
+
+    /** Return formatted table */
+    private function chunkReduceTable(array $members, int $cols, int $depth): string
+    {
+        $trPadding = str_repeat(' ', $depth + 2);
+        return array_reduce(
+                array_chunk(
+                    $members,
+                    $cols,
+                ),
+                fn (string $carry, array $entry) => $carry . $trPadding . "<tr>\n" . implode('', $entry) . $trPadding . "</tr>\n",
+                ''
+            )
+            . str_repeat(' ', $depth + 1) . "</tbody>\n" . str_repeat(' ', $depth) . "</table>";
```

Comment:
Having the `</tbody>` without any corresponding start tag might be confusing in the future, add a comment or move this around?